app/coffee/infrastructure/ExpressLocals.coffee

```diff
@@ -61,11 +61,18 @@ module.exports = (app, webRouter, apiRouter)->
 
 	webRouter.use (req, res, next)-> 
 
+		cdnBlocked = req.query.nocdn == 'true' or req.session.cdnBlocked
+
+		if cdnBlocked and !req.session.cdnBlocked?
+			ip = req.ip || req.socket?.socket?.remoteAddress || req.socket?.remoteAddress
```

Comment:
I think express has a clever ip method that looks at forwarded headers, etc. req.ip will always be the load balancer I think